packages/contracts/contracts/core/plugin/Plugin.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts/utils/introspection/ERC165.sol";
+
+import {PermissionManager} from "../permission/PermissionManager.sol";
+
+import {IDAO} from "../IDAO.sol";
+import {DaoAuthorizable} from "../component/DaoAuthorizable.sol";
+
+/// @title Plugin
+/// @notice NON-Upgradable Plugin Interface that should be directly deployed with `new`.
+abstract contract Plugin is Initializable, ERC165, DaoAuthorizable {
```

Comment:
We are inherting from  OZs`Initializable` twice. The first time from `DaoAuthorizable` and the second time directly.
Note that inheriting from `AppStorage` makes no sense here and is rightfully not used in the constructor to set the DAO variable.